docs/using.rst

```diff
@@ -381,11 +378,15 @@ be on a different computer.
 Nginx
 -----
 
-In the future, if you're running Nginx you will hopefully be able to use this
-plugin to automatically obtain and install your certificate. The Nginx plugin is
-still experimental, however, and is not installed with certbot-auto_. If
-installed, you can select this plugin on the command line by including
-``--nginx``.
+The Nginx plugin is included in Certbot 0.9.0. It should
+work for most configurations. Because it is alpha code, we recommend backing up Nginx
+configurations before using it (though you can also revert changes to
+configurations with ``certbot --nginx rollback``). You can use it by providing
+the ``--nginx`` flag on the commandline.
```

Comment:
nit: Did you mean to write "command line" or did you purposefully make it one word? I think two words is more common, however, I have no preference. I was just checking to see if this was a typo.